azext_iot/central/providers/user_provider.py

```diff
@@ -104,18 +137,46 @@ def add_email(
         self,
         assignee: str,
         email: str,
-        role: Role,
+        role: str,
+        org_id: str,
         central_dns_suffix=CENTRAL_ENDPOINT,
-    ) -> dict:
+    ) -> central_services.user.User:
+        if not email:
+            raise CLIError("Must specify --email when adding a user by email")
+
+        if org_id and self._api_version == ApiVersion.v1_1_preview.value:
+            roles = rf"{org_id}\{role}"
+        else:
+            roles = role
+
+        return central_services.user.addorupdate_email_user(
```

Comment:
create_or_update has the _ but addorupdate does not?

Reply:
+1